public/data/samadpls.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "Abdul Samad",
+  "type": "personal",
+  "bio": "Python Developer | HTML & CSS | CHISEL",
+  "avatar": "https://avatars.githubusercontent.com/u/94792103?v=4",
```

I recommend using the standard and more dynamic link for your profile avatar!!
```suggestion
  "avatar": "https://github.com/samadpls.png",
```

---

Thanks… can u tell me when it will merge??

---

> Thanks… can u tell me when it will merge??

@samadpls Your pull request will only be merged after you've made all changes requested by the reviewers. If you run into any issues, do mention them here :+1: